### PR TITLE
DOC: Lince (Team STAR) 2023 flight sim

### PR DESCRIPTION
**Pull request by LUCKIN13**
## Pull request type

- [x] ReadMe, Docs and GitHub updates

## Checklist

- [x] Docs have been reviewed and added / updated
- [x] Lint (`black rocketpy/ tests/`) has passed locally 
- [x] All tests (`pytest tests -m slow --runslow`) have passed locally
- [x] `CHANGELOG.md` has been updated (if relevant)

## Additional information
Unfortunately I couldn't find any GNSS flight data from the payload. However, this is still a really good example of a simulation model with deployable payload.

### CODE REVIEW

File: docs/examples/lince_flight_sim.ipynb
Comment (Gui-FernandesBR) [inline docs/examples/lince_flight_sim.ipynb:1]: can you double check the plots in the `## Flight Simulation` section please?
Seems odd to me that both 3D plots look the same

Comment (LUCKIN13) [reply docs/examples/lince_flight_sim.ipynb:1]: Double checked all the parameters.  They are indeed pretty similar, but I did some tests and the difference makes sense according to the difference in weight and parachute cds.

File: data/weather/euroc_2023_fixed.nc
Comment (Gui-FernandesBR) [inline data/weather/euroc_2023_fixed.nc:1]: @LUCKIN13  why do you need this file? Isn't there another .nc file for euroc 2023?

**Review by Gui-FernandesBR — COMMENTED** [review]
This plot isn't correct.
The flight should stop at the main deployment height.
You should set a max_time for the first flight object.

image

Also, how did you generate the initial solution for other flights?

**Review by Gui-FernandesBR — CHANGES_REQUESTED** [review]
Also, please add this flight to the comparison plot on the index.rst file

File: data/weather/euroc_2023_fixed.nc
Comment (LUCKIN13) [reply data/weather/euroc_2023_fixed.nc:1]: @Gui-FernandesBR unfortunately, the other file seems to have some kind of error. I noticed, while working on the other examples, that the wind velocities were around 100–200 m/s, which is unrealistic. I made a new request to Copernicus, and I’m considering updating all the EUROC 2023 examples to use the new file and deleting the older one (if it is no longer needed for anything else).

**Comment by LUCKIN13** [comment]
> Also, please add this flight to the comparison plot on the index.rst file

Sorry for forgetting that. Will do that asap

File: data/weather/euroc_2023_fixed.nc
Comment (Gui-FernandesBR) [reply data/weather/euroc_2023_fixed.nc:1]: Are other examples also wrong?

I don't like the name of the new file at all. We should either delete the older or the new file, keeping only one is the best approach.

Comment (LUCKIN13) [reply data/weather/euroc_2023_fixed.nc:1]: I was checking and looks like all the flight examples have wind speeds over 150 m/s. I gave that name knowing that would be temporary (either stick with the old one, or change)

File: docs/examples/lince_flight_sim.ipynb
Comment (Gui-FernandesBR) [reply docs/examples/lince_flight_sim.ipynb:1]: Good.

File: data/weather/euroc_2023_fixed.nc
Comment (Gui-FernandesBR) [reply data/weather/euroc_2023_fixed.nc:1]: I think #735 solved that!

Btw we should create more acceptance tests in order to fix that.

**Comment by Gui-FernandesBR** [comment]
@LUCKIN13 try to read my last commit, I think the #735 solved the problem with the wind speeds.
The problematic code was related to the environment class, not the .nc file, so I've deleted your new file and preserved only the old one.

**Review by Gui-FernandesBR — APPROVED** [review]
LGTM

**Comment by LUCKIN13** [comment]
> @LUCKIN13 try to read my last commit, I think the #735 solved the problem with the wind speeds. The problematic code was related to the environment class, not the .nc file, so I've deleted your new file and preserved only the old one.

Great! Makes sense, since all the other examples also add some problems. Thanks a lot for your help!

**Pull request merged**